src/components/graph/graph.renderer.jsx

```diff
@@ -194,9 +241,21 @@ function renderGraph(
   transform
 ) {
   return {
-    nodes: _renderNodes(nodes, nodeCallbacks, config, highlightedNode, highlightedLink, transform, linksMatrix),
-    links: _renderLinks(nodes, links, linksMatrix, config, linkCallbacks, highlightedNode, highlightedLink, transform),
-    defs: _memoizedRenderDefs(config),
+    nodes: _renderNodes(nodes, nodeCallbacks, config, highlightedNode, highlightedLink, transform.k, linksMatrix),
+    links: _renderLinks(
+      nodes,
+      links,
+      linksMatrix,
+      config,
+      linkCallbacks,
+      highlightedNode,
+      highlightedLink,
+      transform.k
+    ),
+    defs: {
+      ..._memoizedRenderDefs(config),
+      ..._gridLineDefs(config, transform),
```

Comment:
Specifically not generating these inside of `_memoizedRenderDefs` because they have to be re-generated on every draw to account for transforms